scripts/install/install.sh

```diff
@@ -42,6 +42,24 @@ fi
 scripts/install/frontend.sh
 scripts/install/api_key.sh
 
+echo "MACHINE LEARNING WORKER NOTE: Next, you will be asked to install some wordlists for the natural language processing parts of Augur.  Please be advised that the machine learning workers, including the clustering worker, discourse worker, messsage_analysis_worker, pull_request_analysis_worker, and the LSTM model of the insight_worker will not function fully without these lists."
+
+read -r -p "Would you like to install required NLTK word lists for machine learning workers? [Y/n] " response
+case "$response" in
+  [yY][eE][sS]|[yY])
+    echo "Installing... aritificial intelligence, machine learning. deep learning, and crocheting libraries and wordlists"
+    echo "Please checkout https://www.ravelry.com/account/login while you wait. Sure, you may not knit yet, but you know you want to."
+    sleep 2
```

Comment:
Why is there a sleep command here?

Reply:
To give people time to write down the ravelry.com website. I should not attempt humor in code. I will remove.

Reply:
removed.